intel_extension_for_transformers/llm/runtime/graph/models/llama/llama.h

```diff
@@ -29,7 +29,7 @@ enum llama_model {
 static const model_scratch llama_mem_req(int n_layers) {
   switch (n_layers) {
     case 32:
-      return {512ull * MB, 512ull * MB, 1026ull * MB};
+      return {1024ull * MB, 1024ull * MB, 1026ull * MB};
     case 40:
       return {512ull * MB, 512ull * MB, 1608ull * MB};
```

Comment:
why not update other configurations?
does this only happen in mistral?

Reply:
Yes,only happen in mistral.

Reply:
there are no mistral with 40 layers?
does llama-7b encounter the same issue?

Reply:
mistral-7b is only 32 layers, no others mistral,llama-7b has no the same issue,here is test report:
https://[inteltf-jenk.sh.intel.com/job/nlp-toolkit-validation-nightly/375/artifact/deploy/engine/llama-2-7b-chat/llama-2-7b-chat-q4_j_i8_g128-56-1-2012-32.log](https://inteltf-jenk.sh.intel.com/job/nlp-toolkit-validation-nightly/375/artifact/deploy/engine/llama-2-7b-chat/llama-2-7b-chat-q4_j_i8_g128-56-1-2012-32.log)

Reply:
can you look into that why only mistral encounter this issue since they share almost the same architecture?

Reply:
I find a parameter "intermediate_size" in mistral-7b is 14336 and llama2-7b is 11008, maybe this leads to more memory used?

Reply:
confirm it, including which tensor it impacts and how much buffer it need exactly.